### ISSUE TITLE
Media queries are being prefixed

### ISSUE THREAD

**Issue by rolfen** [gasolin/postcss-bidirection#15]
I have a CSS file which starts with a media query.
In the output, the first media query declaration is prefixed (`html[dir=...]`) by postcss-bidirection, which renders it invalid, with all it's declarations.
Strangely, the subsequent two media queries in the CSS file are not being prefixed.
postcss-bidirection should not be prepending selectors to media query declarations.

**Comment by rolfen** [comment]
I have also observed it inserting a `dir` selector before comments at the beginning of the file. It may be doing that when there is no bidirectional match in the file (nothing to insert) so it inserts a selector using `raws.before` which adds it at the beginning of the output file. I am not fully sure of what is happening (I'm new to postCss), however I rewrote a small part of the code and it seems to work now.

I have yet to run the tests.

**Original code**
```
                // overwrite rtlRule.raws.before since its been lazy evaluated
                item.rtlRule.raws.before = '\n\n[dir="ltr"] ';

                // multiple selectors in a rule
                let rtlSelectors = item.rtlRule.selector.split(PATTERN);
                if (rtlSelectors.length) {
                    item.rtlRule.selector =
                      rtlSelectors.join(',\n[dir="ltr"] ');
                }
```

**New code**
```
                // prefix each comma-separated selector
                item.ltrRule.selector = item.ltrRule.selector
                    .split(PATTERN)
                    .map(function(selector){
                        return ("[dir=\"ltr\"] " + selector);
                    })
                    .join(',\n');
```

**Comment by rolfen** [comment]
Actually that only partially fixes the issue. Now two block of rules are moved to the beginning of the file (`rtl` and `ltr`). The other overrides blocks are printed at the right place (after the original block). I'm not sure what triggers this issue, I'll try to investigate.

**Comment by rolfen** [comment]
`root.index(item.rule)` returns -1 when the the rule is inside a `media query` block.
So this happens for rules inside a media query block.

**Comment by rolfen** [comment]
Fixed!

**Old**
```
root.insertAfter(item.rule, item.ltrRule);
```
**New**
```
item.rule.parent.insertAfter(item.rule, item.ltrRule);
```

[Reference](http://api.postcss.org/Node.html#parent)

**Comment by gasolin** [comment]
Thanks! I'll find a time to take a look on this :+1:

**Comment by rolfen** [comment]
You're welcome! I'm planning to make a pull request as soon as I can get it to pass the tests.

**Comment by rolfen** [comment]
The tests fail at your commit `f75d90a67fd3ccbec48e4bb015cc6810185d5a08`, could you have a look?

I don't think there is anything wrong with my code, let me know if you'd like me to do a pull request for this branch:
https://github.com/rolfen/postcss-bidirection/tree/support-media-queries

Cheers!

**Comment by gasolin** [comment]
a pull request will be nice!

**Comment by gasolin** [comment]
border-* properties issue is fixed in 5e5e95e , thanks for report!
